machine/quanta/quanta_bwde/rootconf/sysroot-lib-onie/init-platform

```diff
@@ -0,0 +1,33 @@
+# -*- shell-script -*-
+
+#  Copyright (C) 2014 Curt Brune <curt@cumulusnetworks.com>
+#  Copyright (C) 2017 Audi Hsu <audi.hsu@quantatw.com>
+#
+#  SPDX-License-Identifier:     GPL-2.0
+
+# This script fragment is sourced by /etc/init.d/init-arch.sh, which
+# in turn executes the init_platform_pre_arch() and
+# init_platform_post_arch() functions.
+
+# A machine can use this feature to run any early boot scripts needed
+# by the machine.  A typical usage would be to initialize specific
+# hardware devices.
+
+# Use this function to perform any initializations required by the
+# architecture specific initialization.  This function executes before
+# the architecture initializations.
+init_platform_pre_arch()
+{
+    echo "Running init_platform_pre_arch..."
+    echo "Mount devtmpfs..."
+    mount -t devtmpfs devtmpfs /dev
+    mkdir /dev/pts
+    mount -t devpts devpts /dev/pts
```

Comment:
I played around a bit enabling devtmpfs by default in the kernel and mounting it at boot time.  This ignores all the devices that the make-devices.pl script creates at build time.

It looks promising.  I was also able to remove a bunch of the dynamic device creation code from /etc/init.d/makedev.sh.

This was only with the kvm_x86_64, so it would need quite a bit more testing on other CPU architectures and devices to become the ONIE default.  Kernel configs would also need to be updated to enable the devtmpfs feature.

I won't have a chance to look into this for a while.  It is a long holiday weekend in the US.